conda-recipe/meta.yaml

```diff
@@ -45,24 +42,6 @@ test:
   imports:
     - sdc
 
-outputs:
```

Comment:
Why to remove 'outputs' section at all?

Reply:
One of the purpose of this PR is to align wheels build procedure with others packages.
Here was used conda output - it does not suits for us due to we cannot set additional arguments for wheels build.
That is why output section is removed and wheel build lines are added in build.sh and bld.bat